version.py

```diff
@@ -78,4 +102,8 @@ def get_version(prefix='', cwd='.'):
 
 
 if __name__ == "__main__":
-    print(get_version(prefix='v'))
+    if '--no-update-version-file' in sys.argv:
+        update_version_file = False
+    else:
+        update_version_file = True
```

Comment:
This could be shortened to
``` python
update_version_file = ('--no-update-version-file' in sys.argv')
```

Reply:
Good to know:) Done
@cameel ready for review